<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO" time="0.026" tests="11" errors="0" skipped="0" failures="11">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="C:\Program Files\Java\jdk1.7.0_25\jre\bin"/>
    <property name="java.vm.version" value="23.25-b01"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=";"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.script" value=""/>
    <property name="user.country" value="BR"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="D:\Projetos\contpatri-022013\Desenvolvimento\contpatri-mobile"/>
    <property name="java.runtime.version" value="1.7.0_25-b17"/>
    <property name="java.awt.graphicsenv" value="sun.awt.Win32GraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="C:\Program Files\Java\jdk1.7.0_25\jre\lib\endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="C:\Users\ROGERIO\AppData\Local\Temp\"/>
    <property name="line.separator" value="&#10;"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="user.variant" value=""/>
    <property name="os.name" value="Windows 8"/>
    <property name="classworlds.conf" value="D:\Projetos\.metadata\.plugins\org.eclipse.m2e.launching\launches\m2conf3695544364988584408.tmp"/>
    <property name="sun.jnu.encoding" value="Cp1252"/>
    <property name="java.library.path" value="C:\Program Files\Java\jdk1.7.0_25\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x64;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Git\cmd;D:\Softwares\Maven\bin;C:\Program Files (x86)\Android\android-sdk\tools;C:\Program Files (x86)\Android\android-sdk\platform-tools;C:\Program Files\TortoiseSVN\bin;C:\Program Files (x86)\Android\android-sdk\build-tools;D:\Softwares\Sonar Runner\bin;C:\Program Files\TortoiseGit\bin;."/>
    <property name="m2eclipse.workspace.state" value="D:\Projetos\.metadata\.plugins\org.eclipse.m2e.core\workspacestate.properties"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="51.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="6.2"/>
    <property name="user.home" value="C:\Users\ROGERIO"/>
    <property name="user.timezone" value="America/Sao_Paulo"/>
    <property name="java.awt.printerjob" value="sun.awt.windows.WPrinterJob"/>
    <property name="java.specification.version" value="1.7"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="user.name" value="ROGERIO"/>
    <property name="java.class.path" value="D:\Softwares\Maven\boot\plexus-classworlds-2.4.jar"/>
    <property name="java.vm.specification.version" value="1.7"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="C:\Program Files\Java\jdk1.7.0_25\jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher -B -U -s D:\Softwares\Maven\conf\settings.xml clean install android:deploy android:run"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="pt"/>
    <property name="awt.toolkit" value="sun.awt.windows.WToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.7.0_25"/>
    <property name="java.ext.dirs" value="C:\Program Files\Java\jdk1.7.0_25\jre\lib\ext;C:\Windows\Sun\Java\lib\ext"/>
    <property name="sun.boot.class.path" value="C:\Program Files\Java\jdk1.7.0_25\jre\lib\resources.jar;C:\Program Files\Java\jdk1.7.0_25\jre\lib\rt.jar;C:\Program Files\Java\jdk1.7.0_25\jre\lib\sunrsasign.jar;C:\Program Files\Java\jdk1.7.0_25\jre\lib\jsse.jar;C:\Program Files\Java\jdk1.7.0_25\jre\lib\jce.jar;C:\Program Files\Java\jdk1.7.0_25\jre\lib\charsets.jar;C:\Program Files\Java\jdk1.7.0_25\jre\lib\jfr.jar;C:\Program Files\Java\jdk1.7.0_25\jre\classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="D:\Softwares\Maven"/>
    <property name="file.separator" value="\"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="maven.bootclasspath" value="D:\Softwares\Maven\boot\plexus-classworlds-2.4.jar"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.desktop" value="windows"/>
    <property name="sun.cpu.isalist" value="amd64"/>
  </properties>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.009">
    <failure message="Exception in constructor: testAbrirConexao (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testAbrirConexao (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.002">
    <failure message="Exception in constructor: testGetTodos (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testGetTodos (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.002">
    <failure message="Exception in constructor: testAlterar (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testAlterar (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.002">
    <failure message="Exception in constructor: testTombamentoDAO (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testTombamentoDAO (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.001">
    <failure message="Exception in constructor: testLocalizar (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testLocalizar (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.001">
    <failure message="Exception in constructor: testGetTodosJson (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testGetTodosJson (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.002">
    <failure message="Exception in constructor: testDeletar (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testDeletar (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.002">
    <failure message="Exception in constructor: testGetConexao (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testGetConexao (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.001">
    <failure message="Exception in constructor: testInserir (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testInserir (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.002">
    <failure message="Exception in constructor: testFecharConexao (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testFecharConexao (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.002">
    <failure message="Exception in constructor: testAndroidTestCaseSetupProperly (java.lang.RuntimeException: Stub!&#10;	at android.test.AndroidTestCase.&lt;init&gt;(AndroidTestCase.java:5)&#10;	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.&lt;init&gt;(TesteTombamentoDAO.java:10)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#10;	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)&#10;	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#10;	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:63)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)&#10;	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:132)&#10;	at org.junit.internal.runners.JUnit38ClassRunner.&lt;init&gt;(JUnit38ClassRunner.java:72)&#10;	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#10;	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#10;	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)&#10;	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: Exception in constructor: testAndroidTestCaseSetupProperly (java.lang.RuntimeException: Stub!
	at android.test.AndroidTestCase.<init>(AndroidTestCase.java:5)
	at br.ufg.inf.es.fs.contpatri.mobile.tombamento.TesteTombamentoDAO.<init>(TesteTombamentoDAO.java:10)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at junit.framework.TestSuite.createTest(TestSuite.java:63)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:310)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:153)
	at junit.framework.TestSuite.<init>(TestSuite.java:132)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:72)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:11)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:262)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
)
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:100)
]]></failure>
  </testcase>
</testsuite>